File: oop/java/src/Lista01/Ex03/Ex03.java
package Lista01.Ex03;

import Lista01.KeyboardInput;

public class Ex03 {
    public void printTree(Integer height) { for (; height > 0; height--) System.out.println("*".repeat(height)); }

    public static void main(String[] args) {
        Ex03 treePrinter = new Ex03();
        KeyboardInput keyboardInput = new KeyboardInput(System.in);

        System.out.println("Provide the tree high: ");
        Integer input = keyboardInput.readPositiveInteger();

        treePrinter.printTree(input);
    }
}

File: objetos/lista01_py/Ex03.py
from KeyboardInput import KeyboardInput

class Ex03:
    def printTree(self, height) -> None:
        while height > 0:
            print("*" * height)
            height -= 1

if __name__ == "__main__":
    treePrinter = Ex03()
    keyboardInput = KeyboardInput()

    treeHeight = keyboardInput.readPositiveInt("Provide the tree high: ")
    treePrinter.printTree(treeHeight)